src/getMovedFile.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function getMovedFile(source, destination) {
+  if (!fs.existsSync(destination)) {
+    console.error(`Destination directory ${destination} does not exist.`);
+    process.exit(1);
+  }
```

Comment:
If I'm not mistaken if we don't have this directory, that would mean, the user wants to just rename the source file.
![image](https://user-images.githubusercontent.com/63786090/223655792-9976c9ef-6f5e-4427-a932-7d18c70ea850.png)
As a suggestion, we can check the 'destination' for existing and for is it a directory or just will be a file.
```suggestion
 const destPath = path.join(__dirname, destination);
  let isDestinationDir;

  try {
    isDestinationDir = fs.statSync(destPath).isDirectory();
  } catch (error) {
    isDestinationDir = false;
  }

  const destinationPath = isDestinationDir
    ? path.join(destPath, path.basename(source))
    : destPath;
```
So, by checking `destPath` in try block for existing we are doing two things at once:
1) Checks if it's an existing folder, if yes, method `isDirecotry()` will return true otherwise false.
2) Whether exists anything, if no, `statSync` method will throw an error, in that case, we understand that we just have to rename a file.